nml/expression/storage_op.py

```diff
@@ -111,6 +111,10 @@ def reduce(self, id_dicts=None, unknown_id_fatal=True):
             raise generic.ScriptError("Register to access must be an integer.", register.pos)
         if isinstance(register, ConstantNumeric) and register.value > self.info["max"]:
             raise generic.ScriptError("Maximum register for {} is {:d}".format(self.name, self.info["max"]), self.pos)
+        if isinstance(register, ConstantNumeric) and 0x80 <= register.value <= 0xFF:
```

Comment:
I think this needs a "not self.info["perm"]", otherwise it triggers for persistent storage.
Though maybe it's also smart to extend "self.info" with some "reserved" member or something.